vllm/model_executor/models/gemma.py

```diff
@@ -325,15 +325,23 @@ def load_weights(self,
                 if shard_name not in name:
                     continue
                 name = name.replace(shard_name, param_name)
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
                 param = params_dict[name]
                 weight_loader = param.weight_loader
                 weight_loader(param, loaded_weight, shard_id)
                 break
             else:
+                # Skip loading extra bias for GPTQ models.
+                if name.endswith(".bias") and name not in params_dict:
+                    continue
                 # GemmaRMSNorm is different from Llama's in that it multiplies
                 # (1 + weight) to the output, instead of just weight.
                 if "norm.weight" in name:
                     loaded_weight += 1.0
+                if name == "lm_head.weight" and name not in params_dict:
```

Comment:
Quick question: Why this check is needed?

Reply:
Gemma's input and output weights are shared and there is no separate `lm_head` parameter.
There will be no problem when loading the original model, but when loading the quantized model, `lm_head` is not included in `params_dict`. I skip it to load the model normally.

Reply:
After reinstalling from the latest main branch of transformers and AutoGPTQ, I found that removing them works fine. Thanks for testing!